debian/changelog

```diff
@@ -0,0 +1,5 @@
+golang-logrus (0.6.0) UNRELEASED; urgency=medium
```

Comment:
Oh, this has no "Debian revision"!  This needs to be `0.6.0-1`, since it's the first Debian release of `0.6.0` (which is the upstream version).

See https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-Version (Debian Policy § 5.6.12) for more information about versions. :+1: